internal/controllers/inventory_controller.go

```diff
@@ -174,8 +174,17 @@ func (t *reconcilerTask) setupResourceServerConfig(ctx context.Context, defaultR
 		return
 	}
 
+	internalPort := int32(0)
```

Comment:
We could consider moving these to their own function since it's used in 3 places in this file.

Reply:
fixed